.devcontainer/postCreateCommand.sh

```diff
@@ -0,0 +1,19 @@
+mkdir -p $HOME/.kube
```

Comment:
`#!/bin/bash`

since by default we use Bash it makes sense to add shebang to avoid issues